_posts/2018-03-12-nunit-310.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "NUnit 3.10.1 Released"
+date:   2018-03-12 12:00:00 -0000
+categories: news update nunit
+---
+This release adds a .NET Standard 2.0 version of the framework which re-enables most of the features that have been missing in our earlier .NET Standard builds like parallelism, timeouts, directory and path based asserts, etc. It also contains numerous bug fixes and smaller enhancements. We've improved our XML docs, fixed performance issues and added more detail to Multiple Asserts.
+
+This release also contains source linked PDB files allowing developers to debug into the NUnit Framework. This allows you to track down errors or see how the framework works.
+
+In order to support the .NET Standard 2.0 version, the NUnit project switched to the new CSPROJ format and now requires Visual Studio 2017 to compile. This only effects people contributing to the project. NUnit still supports building and compiling your tests in older .NET IDE's and NUnit still supports older versions of the .NET Framework back to 2.0. For contributors, NUnit can now compile all supported targets on Windows, Linux and Mac using the Cake command line build.
```

Comment:
This doesn't yet have the two changes that the framework repo has.

Reply:
Thought I did that. I will deal with it in the morning. Thanks.